appcelerator/titanium_mobile
# [TIMOB-16989] 3_3_X iOS: Updating AnalyticsModule to work with APSAnaltyics module

Need to move out the libAnalytics.a from the SDK needs to be pulled in during build process. Will update that in the next update.

TESTING INSTRUCTIONS
build a new app with following code. 
Run the code in JIRA Make sure nothing crashes.
